Neos.Neos/Classes/Command/SiteCommandController.php

```diff
@@ -190,6 +199,58 @@ public function importCommand(string $packageKey = null, string $path = null, st
         $this->siteImportService->importFromPath($contentRepositoryId, $path, $onProcessor, $onMessage);
     }
 
+    /**
+     * Export sites
+     *
+     * This command allows to export all current sites.
+     *
+     * !!! At the moment always all sites are exported. This will be improved in future!!!
+     *
+     * If a path is specified, this command expects the corresponding directory to contain the exported files
+     *
+     * If a package key is specified, this command expects the export files to be located in the private resources
+     * directory of the given package (Resources/Private/Content).
+     *
+     * @param string|null $packageKey Package key specifying the package containing the sites content
+     * @param string|null $path relative or absolute path and filename to the export files
+     * @return void
+     */
+    public function exportCommand(string $packageKey = null, string $path = null, string $contentRepository = 'default', bool $verbose = false): void
+    {
+        $exceedingArguments = $this->request->getExceedingArguments();
+        if (isset($exceedingArguments[0]) && $packageKey === null && $path === null) {
+            if (file_exists($exceedingArguments[0])) {
+                $path = $exceedingArguments[0];
+            } elseif ($this->packageManager->isPackageAvailable($exceedingArguments[0])) {
+                $packageKey = $exceedingArguments[0];
+            }
+        }
+        if ($packageKey === null && $path === null) {
+            $this->outputLine('<error>You have to specify either <em>--package-key</em> or <em>--filename</em></error>');
+            $this->quit(1);
+        }
```

Comment:
nitpick: this is the same code as for `importCommand`, could be extracted to a private method

Reply:
adjusted